internal/pkg/xds/xds_mcp.go

```diff
@@ -130,81 +128,81 @@ func (adss *adsServer) push(resources []*anypb.Any) error {
 }
 
 // pushToSubscribers pushes MCP resources to active subscribers.
-func (adss *adsServer) pushToSubscribers() error {
-	var mcpResources []*anypb.Any
-	for _, host := range []string{"example.com", "istio.io"} {
-		se, err := makeMCPServiceEntry(host)
-		if err != nil {
-			return fmt.Errorf("failed to create MCP service entry for the hostname %s: %v", host, err)
-		}
-		mcpResources = append(mcpResources, se)
-	}
-
-	adss.subscribers.Range(func(key, value any) bool {
-		log.Print("Sending to subscriber ", fmt.Sprintf(subIDFmtStr, key.(uint64)))
-
-		if err := value.(*subscriber).stream.Send(&discovery.DiscoveryResponse{
-			TypeUrl:     "networking.istio.io/v1alpha3/ServiceEntry",
-			VersionInfo: strconv.FormatInt(time.Now().Unix(), 10),
-			Resources:   mcpResources,
-			ControlPlane: &envoycfgcorev3.ControlPlane{
-				Identifier: os.Getenv("POD_NAME"),
-			},
-		}); err != nil {
-			log.Print("Error sending MCP resources: ", err)
-			value.(*subscriber).closeStream()
-			adss.subscribers.Delete(key)
-		}
-
-		return true
-	})
-
-	return nil
-}
+// func (adss *adsServer) pushToSubscribers() error {
+// 	var mcpResources []*anypb.Any
+// 	for _, host := range []string{"example.com", "istio.io"} {
+// 		se, err := makeMCPServiceEntry(host)
+// 		if err != nil {
+// 			return fmt.Errorf("failed to create MCP service entry for the hostname %s: %v", host, err)
+// 		}
+// 		mcpResources = append(mcpResources, se)
+// 	}
+
+// 	adss.subscribers.Range(func(key, value any) bool {
+// 		klog.Infof("Sending to subscriber %s", fmt.Sprintf(subIDFmtStr, key.(uint64)))
+
+// 		if err := value.(*subscriber).stream.Send(&discovery.DiscoveryResponse{
+// 			TypeUrl:     "networking.istio.io/v1alpha3/ServiceEntry",
+// 			VersionInfo: strconv.FormatInt(time.Now().Unix(), 10),
+// 			Resources:   mcpResources,
+// 			ControlPlane: &envoycfgcorev3.ControlPlane{
+// 				Identifier: os.Getenv("POD_NAME"),
+// 			},
+// 		}); err != nil {
+// 			klog.Errorf("Error sending MCP resources: %v", err)
+// 			value.(*subscriber).closeStream()
+// 			adss.subscribers.Delete(key)
+// 		}
+
+// 		return true
+// 	})
+
+// 	return nil
+// }
```

Comment:
@matteo-casalino can you remove commented code? We can always look at previous commits or pull requests to reuse old code.